meson_options.txt

```diff
@@ -0,0 +1,9 @@
+option('paranoid'    , type: 'boolean', value: false    , description: 'Add file and line number to WARNING messages.')
```

Comment:
Default should be file and line number. Paranoid with autotools sets -Werror which means the game will error out at the first warning.